app/assets/stylesheets/omni_nav/omni_nav_v1/modules/omni-nav.scss

```diff
@@ -325,47 +338,58 @@ a.cu_nav_button {
     z-index: 99100;
   }
 
+  // Container surrounding input.gsc-input
+  td.gsib_a {
+    padding: 0 0 0 5px;
+  }
+
   // The query input box
   input.gsc-input {
-    @include reset-box-model();
-
     width: 100%;
-    padding: 6px 0 3px;
-    border-bottom: 1px solid $grey2;
+    margin: 0;
+    text-indent: 0px !important;
```

Comment:
In this situation, the use of `!important` is absolutely essential, even though it's a big code smell. That's because Google added more inline styles to their custom search code, which is an even bigger code smell in my opinion.